packages/redux-saga-web3-erc20/src/erc20/actions.js

```diff
@@ -1,7 +1,41 @@
-import { createMethodCall } from "redux-saga-web3-eth-contract";
+import {
+  createMethodCall,
+  createMethodSend,
+} from "redux-saga-web3-eth-contract";
+import { CONTRACT_NAME } from "./constants";
 
-function balanceOf(who, options = {}) {
-  return createMethodCall("ERC20", "balanceOf", options)(who);
+// Implements actions interface for ERC20: https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md
+
+export function name(options = {}) {
+  return createMethodCall(CONTRACT_NAME, "name", options)();
+}
+
+export function symbol(options = {}) {
+  return createMethodCall(CONTRACT_NAME, "symbol", options)();
+}
+
+export function totalSupply(who, options = {}) {
+  return createMethodCall(CONTRACT_NAME, "totalSupply", options)(who);
+}
+
+export function balanceOf(owner, options = {}) {
+  return createMethodCall(CONTRACT_NAME, "balanceOf", options)(owner);
+}
+
+export function allowance(owner, spender, options = {}) {
+  return createMethodCall(CONTRACT_NAME, "allowance", options)(owner, spender);
+}
+
+export function transfer(to, tokens, options = {}) {
```

Comment:
`transfer`, `approve`, and `transferFrom` should use `createMethodSend` since they are modifying state